scripts/updateTestData.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const path = require('path');
+
+const updateChainsList = () => {
```

Comment:
This is a preparation script that will run for each test run and update the test data for the available chains to have it in a TypeScript object, making it possible to specify this data as an iteration object in tests.